accessibility-oscilloscope/package/oscilloscope-trigger/oscilloscope-trigger.mk

```diff
@@ -0,0 +1,19 @@
+################################################################################
+#
+# oscilloscope-trigger
+#
+################################################################################
+
+OSCILLOSCOPE_TRIGGER_VERSION = v0.0.1
+OSCILLOSCOPE_TRIGGER_SOURCE = $(OSCILLOSCOPE_TRIGGER_VERSION).tar.gz
+OSCILLOSCOPE_TRIGGER_SITE = https://github.com/accessibility-oscilloscope/oscilloscope-trigger/archive/refs/tags
+
+define OSCILLOSCOPE_TRIGGER_BUILD_CMDS
+	$(MAKE) $(TARGET_CONFIGURE_OPTS) -C $(@D) oscilloscope-trigger
```

Comment:
why are we running `make` for a python script?

Reply:
for clarity, this stage (`BUILD_CMDS`) is only needed if you're actually... building anything. for python, likely all you want is to just copy over the file into `/usr/bin/` or whatever.

Reply:
Ah my bad. I had assumed that the make was a part of the process for buildroot where it built a package that was to be used in the buildroot but had not realized it doesn't apply to scripts. I'll make modifications based upon the findings Jack has from previous work in the other signal processing king repository.